app/collins/models/RemoteAssetFinder.scala

```diff
@@ -279,7 +280,8 @@ object RemoteAssetFinder {
     val key = searchParams.paginationKey + clients.map { _.tag }.mkString("_")
     val stream = Cache.getAs[RemoteAssetStream](key).getOrElse(new RemoteAssetStream(clients, searchParams))
     val results = stream.slice(pageParams.page * pageParams.size, (pageParams.page + 1) * (pageParams.size))
-    Cache.set(key, stream, 30)
+    val timeout = MultiCollinsConfig.queryCacheTimeout
```

Comment:
This code path will never be hit if multi-collins isn't enabled. In order to do a multi-collins query `MultiCollinsConfig.enabled` must be true and the `remoteLookup` parameter specified: https://github.com/tumblr/collins/blob/master/app/collins/controllers/actions/asset/FindAction.scala#L45

I can't really think of a case where we'd want to cache results from different instances for different amounts of time. I think for the scope of this PR i'd like to leave that out. We'd probably have to store the timeout in the datacenter asset which makes things a bit more complicated.